include/Zydis/DecoderTypes.h

```diff
@@ -108,6 +108,82 @@ typedef enum ZydisMemoryOperandType_
 /* Decoded operand                                                                                */
 /* ---------------------------------------------------------------------------------------------- */
 
+/**
+ * Extended info for register-operands.
+ */
+typedef struct ZydisDecodedOperandReg_ {
+    /**
+     * The register value.
+     */
+    ZydisRegister value;
+} ZydisDecodedOperandReg;
+/**
```

Comment:
We should add some linebreaks (before all the struct comments).

Reply:
I agree, I wasn't sure if I should do it or not since I copied it out as is, I addressed that.

Reply:
Thanks a lot!